sections/advanced/existing-css.js

```diff
@@ -86,6 +86,41 @@ const ExistingCSS = () => md`
   This technique takes advantage of styled-components’ server-side rendering (SSR) support. At
   runtime, styled-components searches the document for existing tags and recycles them. Any new
   \`<style>\` tags are inserted immediately after the existing ones.
+
+  ### Avoiding Conflicts When Embedded
```

Comment:
Perhaps this could be reworded to "Avoiding Conflicts With Existing Page Styles"?

Reply:
Hmm. I'm not sure that would clearly delineate this section from the preceding section, "Issues with Specificity." This section is specifically about issues that arise when styled-components is on a page that you don't control, so that e.g. there could be style rules you don't know about, and there could be another instance of styled-components.

I realize that the term "embedding" is overloaded with meanings, so I'd welcome other suggestions.

Reply:
"Using Styled Components on Third Party Sites" possibly?

Reply:
Not sure that's more accurate. It could be your own site but with third-party CSS or JavaScript, for instance. Of course, "embedded" isn't accurate in that scenario, either.

How about "Avoiding Conflicts with Third-Party Styles and Scripts"?